### DISCUSSION TITLE
how to disable all message without waiting

### DISCUSSION THREAD

**Discussion by nbari** [folke/noice.nvim#729] [Q&A] [answered]
I  installed via mason `vetur-vls`, which works but for known reasons I get a huge message when opening any `*.vue` file:

![S5t9nuv](https://github.com/folke/noice.nvim/assets/364199/1be4382f-dbf7-41ae-920d-ba7e420e3ce0)

Therefore, I am wondering if there is a key binding or way to disable the messages to prevent blocking, some times I also get other error messages but after some seconds they go away, is there a way to remove them all quickly?

**Comment by Allaman** [comment, marked as answer]
Hmm, I think you should look at the root cause why vetur-vls does notify you.

However, you might want to take a look at routes like in [this example](https://github.com/folke/noice.nvim/wiki/Configuration-Recipes#hide-written-messages). Just adjust this snippet to match the pattern of the notifications you want to get rid of.

Another option might be to run (or bind to a mapping) `:Noice dismiss`

edit: https://github.com/folke/noice.nvim/discussions/687
